### PR TITLE
Feature/events

### PR DESCRIPTION
**Pull request by leonardodiasb**
This PR I: 
- Created a folder and copied the calculate and operate files.
- Set a state to the Calculator constructor.
- Created a click event and passed the calculator functionality to it.
- Assigned the events to all buttons.
- Created an if statement conditional to avoid bugs when double clicking on operation buttons.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -3,45 +3,56 @@ import React, { Component } from 'react';
 import './Calculator.css';
 import Button from './Button';
 
+import calculate from './logic/calculate';
+
```
Comment (DeliceLydia) [inline src/components/Calculator.js]: - [ ] The Calculator component should import the `calculate.js & operate.js` files from the logic directory but currently, you imported `calculate` only kindly fix that
- [ ] I recommend that you move out `logic` folder outside the component

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @leonardodiasb👏,

Good job so far! your calculator is looking exactly like the design and it's working perfectly..
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



### Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (leonardodiasb) [reply src/components/Calculator.js]: Hello Lydia,
Thanks for the review. The operate class is being used in the calculate.js file which has the only function that I need to apply to the onclick events. If I import the operate, then I will get the "no-unused-vars" linter error.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @leonardodiasb 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**